engine/render/renderer/Renderer.h

```diff
@@ -26,7 +26,7 @@ class Renderer
 {
 public:
 
-    explicit Renderer(Window& window);
+    explicit Renderer(Window& window, const char* defaultFontPath);
```

Comment:
As mentioned before, I don't like this and I'd rather have it display nothing or a test texture instead, otherwise the renderer depends on a font asset existing.

Reply:
See my comment above.